tools/test_value_scraper_vmc.py

```diff
@@ -0,0 +1,168 @@
+        sys.exit()
+
+    output["vcenter_detail"]["vcenter_server"] = get_server_from_url(
+        output["vcenter_detail"]["vcenter_url"]
+    )
+    return output["vcenter_detail"]
+
+
+def update_vmc_vcenter_config(config_file, vcenter_server_detail, args):
+    parser = ConfigParser()
+    parser.read(config_file)
+    parser.set("vmc_connect", "hostname", args.vmc_hostname)
+    parser.set("vmc_connect", "vcenter_hostname", vcenter_server_detail["vcenter_server"])
+    parser.set("vmc_connect", "refresh_key", args.refresh_key)
+    parser.set("vmc_connect", "authorization_host", args.authorization_host)
+    parser.set("vmc_connect", "org_id", args.org_id)
+    parser.set("vmc_connect", "sddc_id", args.sddc_id)
+
+    parser.set("vmc_vcenter_connect", "hostname", vcenter_server_detail["vcenter_server"])
+    parser.set("vmc_vcenter_connect", "username", vcenter_server_detail["username"])
+    parser.set("vmc_vcenter_connect", "password", vcenter_server_detail["password"])
+    # TODO will change this when handling of cert generation is done
+    parser.set("vmc_connect", "verify_ssl", "false")
+    parser.set("vmc_vcenter_connect", "verify_ssl", "false")
+
+    with open(config_file, "w") as cf:
+        parser.write(cf)
+
+
+def do_it(config_file):
+    nsx_reverse_proxy_server = get_nsx_reverse_proxy_server(args)
+    print("******** updating vmc nsx config *********")
+    update_vmc_nsx_config(config_file, nsx_reverse_proxy_server, args)
+    vcenter_server_detail = get_vcenter_server_detail(args)
+    print("******** updating vmc vcenter config *********")
+    update_vmc_vcenter_config(config_file, vcenter_server_detail, args)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-c",
+        dest="create",
+        action="store_true",
+        default=False,
+        help="Create config file if not exists.",
+    )
+    parser.add_argument("CONFIG_FILE", type=Path, help="Path to vmc config file")
+
+    parser.add_argument(
+        "-v",
+        "--vmc_hostname",
+        dest="vmc_hostname",
+        default="stg.skyscraper.vmware.com",
+        help="VMC console for getting sddc details",
+    )
+    parser.add_argument(
+        "-a",
+        "--authorization_host",
+        dest="authorization_host",
+        default="console-stg.cloud.vmware.com",
+        help="cloud console for sddc",
+    )
+    parser.add_argument(
+        "-r", "--refresh_key", dest="refresh_key", help="Developer token for accessing REST API"
+    )
+    parser.add_argument("-o", "--org_id", dest="org_id", help="Organization id")
+    parser.add_argument("-s", "--sddc_id", dest="sddc_id", help="sddc id")
+    args = parser.parse_args()
+
+    config_file = args.CONFIG_FILE
+    print(config_file)
+    if not config_file.is_file():
+        if args.create:
+            source_file = Path(__file__).parent.parent / "tests" / "integration" / "vmc_config.ini"
```

Comment:
**suggestion** this file should be removed from the repository, so it shouldn't be referenced here.

Reply:
done